_posts/2016-05-20-drupalcon-nola-recap.md

```diff
@@ -0,0 +1,118 @@
+
+### Wednesday - Sessions and Lullabot Party
+
+After the presentation was behind me, I was happy to be more involved with the daily activities of Drupalcon, i.e. **attending sessions!**. I went to a handful, but my favorite of the day was [Lessons from Wordpress Core](https://events.drupal.org/neworleans2016/sessions/lessons-wordpress-core). It was a nice discussion juxtaposing Wordpress and Drupal powered websites given by two [Pantheon](https://pantheon.io/) employees who had respective expertise in one of the frameworks. One primary takeaway for me was the discrepancy in approach with Drupal vs. Wordpress in backward compatibility. It is a _very_ complex topic, but in some ways the backwards compatible approach has contributed to Wordpress's high degree of user-loyalty as well as comparatively fast adoption to the newest version. The Drupal leadership is very conscious of this and is having very meaningful conversations around Backwards compatibility. Larry Garrield ([Crell](https://www.drupal.org/u/crell)) [provides great insight to this](https://www.youtube.com/watch?v=JeoCHqzvUFY&feature=youtu.be&t=3296) at the end of the session.
+
+After sessions closed for the day, we had a nice dinner with friends from [DevCollaborative](http://devcollaborative.com/about-us), [Advomatic](https://www.advomatic.com/our-team/sarah-german)
+and made our way on over to the [Lullabot party](https://events.drupal.org/neworleans2016/lullabots-8th-annual-drupalcon-party). The theme of meeting those who we only digitally knew, while making new friends certainly continued at the Lullabot party. It was a great time!
+
+### Thursday - the last day of sessions
+
+Thursday was the final day of sessions, and of course the [Drupal 6 burial party](https://events.drupal.org/neworleans2016/kalamuna-four-kitchens-present-drupal-6-mourning-party) - #SoCheeky. The sessions I attended were all quite pertinent to me. In my talk, I reference the lack of an official release for an e-commerce platform for Drupal 8 is one reason for slowed adoption compared to Drupal 7. The Commerce guys did elicit a lot of applause in [their session](https://events.drupal.org/neworleans2016/sessions/creating-stores-drupal-commerce-drupal-8) about what _is coming soon_ for Drupal Commerce's official Drupal 8 release, but were only willing to commit to an ready by the "end of summer" which left [some people](https://youtu.be/EQxA3VYal54?t=3161) chomping at the bit for an official release that is still at arm's length.
```

Comment:
@chrisarusso - I think you have an extra "an" in here `but were only willing to commit to an ready by the "end of summer" `